modules/mod_comment/mod_comment.erl

```diff
@@ -95,27 +113,43 @@ observe_search_query(_, _Context) ->
 observe_rsc_merge(#rsc_merge{looser_id=LooserId, winner_id=WinnerId}, Context) ->
     m_comment:merge(WinnerId, LooserId, Context).
 
+observe_comment_insert(#comment_insert{comment_id=Id}, Context) ->
+    Comment = m_comment:get(Id, Context),
+    case proplists:get_value(rating, Comment) of
+        undefined -> ok;
+        _Rating ->
+           {rsc_id, RscId} = proplists:lookup(rsc_id, Comment),
+           update_rating_pivot(RscId, z_acl:sudo(Context))
+    end.
+
+update_rating_pivot(RscId, Context) ->
+    Ratings = [R || R <- [proplists:get_value(rating, C) || C <- m_comment:list_rsc(RscId, Context)], R =/= undefined],
```

Comment:
Line 126 is too long:     Ratings = [R || R <- [proplists:get_value(rating, C) || C <- m_comment:list_rsc(RscId, Context)], R =/= undefined],.